CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Contributing Issues
+
+### Before Submitting an Issue
+First, if your issue is a question or you need help with setting up / configuring Streisand, then please ask on the [streisand-discussions](https://github.com/jlund/streisand-discussions/) repo.
+
+If a bug or technical feature request, please do a search in [open issues](https://github.com/jlund/streisand/issues) to see if the issue or feature request has already been filed.

```

Comment:
The start of this sentence reads a little abrupt to me. What about "If you are contributing an issue pertaining to a bug or technical feature request, please do a ..."